src/test/java/com/dougnoel/sentinel/system/FileManagerTests.java

```diff
@@ -35,4 +46,47 @@ public void testPathSeparatorReplacement() {
 
         assertTrue(resultMessage, resultPathString.contains(File.separator) && !resultPathString.contains("/"));
     }
+
+    @Test
+    public void localAppDataTranslation() {
```

Comment:
If this test and the two below run on mac/linux will they fail? Should they be setting the OS to windows?

Reply:
From what I can tell this test class already has a teardown that's resetting the system property to reflect the actual OS of the system. Each of these methods would only need 1 line at the beginning to set the OS property value to "windows" in order to make them run on any platform. But I'm happy to defer to Doug's judgement on this. Just something I thought I should bring up.